src/content/en/2019/markup.md

```diff
@@ -0,0 +1,301 @@
+
+### Common use cases and solutions
+
+What if, for example, we were interested in people solving common use cases?  This could be because we're looking for solutions to use cases that we currently have ourselves, or for researching more broadly what common use cases people are solving with an eye toward incubating some standardization effort.  Let's take a common example: Tabs.  Over the years there have been a lot of requests for things like tabs.  We can use a fuzzy search here and find that there are [many variants of tabs](https://rainy-periwinkle.glitch.me/permalink/c6d39f24d61d811b55fc032806cade9f0be437dcb2f5735a4291adb04aa7a0ea.html).  It's a little harder to count use here since we can't as easily distinguish if two elements appear on the same page, so the count provided there conservatively simply takes the one with the largest count - in most cases the real number of domains is probably significantly larger.
+
+There are also [lots of accordions](https://rainy-periwinkle.glitch.me/permalink/e573cf279bf1d2f0f98a90f0d7e507ac8dbd3e570336b20c6befc9370146220b.html), [dialogs](https://rainy-periwinkle.glitch.me/permalink/0bb74b808e7850a441fc9b93b61abf053efc28f05e0a1bc2382937e3b78695d9.html), at least [65 variants of carousels](https://rainy-periwinkle.glitch.me/permalink/651e592cb2957c14cdb43d6610b6acf696272b2fbd0d58a74c283e5ad4c79a12.html), lots of stuff about ['popups'](https://rainy-periwinkle.glitch.me/permalink/981967b19a9346ac466482c51b35c49fc1c1cc66177ede440ab3ee51a7912187.html), at least [27 variants of toggles and switches](https://rainy-periwinkle.glitch.me/permalink/2e6827af7c9d2530cb3d2f39a3f904091c523c2ead14daccd4a41428f34da5e8.html), and so on.
+
+Perhaps we could research why we need[ 92 variants of button related elements that aren't a native button](https://rainy-periwinkle.glitch.me/permalink/5ae67c941395ca3125e42909c2c3881e27cb49cfa9aaf1cf59471e3779435339.html), for example, and try to fill the native gap.
+
+If we notice popular things pop up (like `<jdiv>`, solving chat) we can take knowledge of things we know (like, that that is what `<jdiv>` is about, or `<olark>`) and try to look [at at least 43 things we've built for tackling that](https://rainy-periwinkle.glitch.me/permalink/db8fc0e58d2d46d2e2a251ed13e3daab39eba864e46d14d69cc114ab5d684b00.html) and follow connections to survey the space.
+
+### In Summary
+
+So, there's lots of data here, but to summarize: 
+
+* Pages have more elements than they did 14 years ago - both on average and max.
+* The lifetime of things on home pages is *very *long.  Deprecating or discontinuing things doesn't make them go away, and it might never.
```

Comment:
```suggestion
* The lifetime of things on home pages is *very* long.  Deprecating or discontinuing things doesn't make them go away, and it might never.
```